### DISCUSSION TITLE
PocketBase now available as a one-click install on Cloudron!

### DISCUSSION THREAD

**Discussion by The-Nils** [pocketbase/pocketbase#4929] [Show and tell]
Hey PocketBase community!

I'm excited to share that , the Cloudron team and me, have packaged PocketBase as a one-click install app on Cloudron, a self-hosting solution that makes it super easy to deploy and manage apps on your servers.

The app is marked as "unstable" until PocketBase reaches its v1 release but at the moement it's working flawlessly 🚀.

I've only recently discovered PocketBase myself, about a week ago, and I've been incredibly impressed with its capabilities. In fact, I've already migrated a couple of my projects to use PocketBase's APIs, and the process has been surprisingly straightforward, thanks to the automatic API documentation generated for each table. The documentation made it a breeze to integrate my frontend applications with the PocketBase backend.

while I'm at it: 
cloudron can be used as SMTP and OpenId provider
- is there a way to set these settings at PocketBase launch for an even more "integrated" Cloudron experience?

**Comment by ganigeorgiev** [comment]
> is there a way to set these settings at PocketBase launch for an even more "integrated" Cloudron experience?

I'm not sure if it will be a good idea to apply this by default for all users, but in general you can initialize default app settings via migration (with [Go](https://pocketbase.io/docs/go-migrations/#initialize-default-application-settings) or [JSVM](https://pocketbase.io/docs/js-migrations/#initialize-default-application-settings)).
